README.md

```diff
@@ -47,28 +47,38 @@ After checking out the mozregression repository from GitHub, this is a two step
 1. Be sure you are using Python 3.6 or above: earlier versions are not supported (if you
```

Comment:
3.6 EOL at the beginning of the year; we really should make this 3.7 at a minimum :)